internal/pkg/skuba/deployments/ssh/kubernetes.go

```diff
@@ -73,7 +73,7 @@ func kubernetesInstallNodePattern(t *Target, data interface{}) error {
 		updatedVersion := kubernetes.MajorMinorVersion(version.MustParseSemantic(kubernetesBaseOSConfiguration.UpdatedVersion))
 		patternName = fmt.Sprintf("patterns-caasp-Node-%s-%s", currentVersion, updatedVersion)
 	}
-	_, _, err := t.ssh("zypper", "--non-interactive", "install", "--force", patternName)
+	_, _, err := t.ssh("zypper", "--non-interactive", "install", "--recommends", "--force", patternName)
```

Comment:
Now that we are at it, do you think it would make sense to remove `--force` here? I know @davidcassany has recommended against using `--force` several times, maybe we could tackle this as well here, @davidcassany WDYT?

Reply:
I would feel better about doing this in a separate PR. I think it warrants a larger talk about why we wanted it there in the first place.

Reply:
I'm fine with another PR, why it was here was mainly because we started off without a pattern (with packages directly) and then we moved to the pattern. When we started using a pattern, @davidcassany has recommended several times to not use `--force`.

Reply:
I will create a new PR today to address this.